merkle/log_proofs_test.go

```diff
@@ -263,6 +263,7 @@ func TestCalcConsistencyProofNodeAddressesBadInputs(t *testing.T) {
 	}
 }
 
+// TODO(pkalinnikov): Remove desc when all the tests use t.Run.
 func comparePaths(t *testing.T, desc string, got, expected []NodeFetch) {
```

Comment:
add a `t.Helper()`?

Reply:
Will do in a follow-up. But more likely it will be deleted in favour of [`cmp.Diff`](https://godoc.org/github.com/google/go-cmp/cmp#Diff).

Reply:
👍